package/features/environment.py

```diff
@@ -87,33 +100,49 @@ def call(cmd, verbose=False):
     # Activate environment
     context.env["PATH"] = path_sep.join(path)
 
-    # install this plugin by resolving the requirements using pip-compile
-    # from pip-tools due to this bug in pip: https://github.com/pypa/pip/issues/988
-    call([context.python, "-m", "pip", "install", "-U", "pip", "pip-tools"])
-    pip_compile = str(bin_dir / "pip-compile")
-    with tempfile.TemporaryDirectory() as tmpdirname:
-        compiled_reqs = Path(tmpdirname) / "compiled_requirements.txt"
-        call([pip_compile, "--output-file", str(compiled_reqs), "requirements.txt"])
-        call([context.pip, "install", "-r", str(compiled_reqs)])
+    call(
+        [
+            context.python,
+            "-m",
+            "pip",
+            "install",
+            "-U",
+            "pip>=20.0",
+            "setuptools>=38.0",
+            "wheel",
+        ],
+        env=context.env,
+    )
 
     for wheel_path in glob.glob("dist/*.whl"):
         os.remove(wheel_path)
-    call([context.python, "setup.py", "clean", "--all", "bdist_wheel"])
+    call([context.python, "setup.py", "clean", "--all", "bdist_wheel"], env=context.env)
+
+    call([context.pip, "install", "-U"] + glob.glob("dist/*.whl"), env=context.env)
```

Comment:
I noticed we have this inconsistency in kedro as well, not sure why we call `python -m pip` in once case and `pip` directly in the other.

Reply:
Hmm yeah good point. I'm not sure either why we do that. I'll change it here to be consistent.